artichoke-backend/src/extn/core/time/mruby.rs

```diff
@@ -112,11 +112,15 @@ unsafe extern "C" fn time_self_now(mrb: *mut sys::mrb_state, _slf: sys::mrb_valu
 }
 
 unsafe extern "C" fn time_self_at(mrb: *mut sys::mrb_state, _slf: sys::mrb_value) -> sys::mrb_value {
-    let (seconds, microseconds) = mrb_get_args!(mrb, required = 1, optional = 1);
+    let (seconds, opt1, opt2, opt3) = mrb_get_args!(mrb, required = 1, optional = 3);
```

Comment:
Could quite possibly be done with args_rest, but this will get a more strict error (and more in line with MRI) if I'm not mistaken.

Ref: https://ruby-doc.org/core-2.6.3/Time.html#method-c-at